drivers/disk/nvme/Kconfig

```diff
@@ -37,6 +37,21 @@ config NVME_IO_ENTRIES
 	  This sets the amount of allocated IO queue entries.
 	  Do not touch this unless you know what you are doing.
 
+config NVME_RETRY_COUNT
+	int "Retry count"
+	default 2
+	help
+	  This sets the amount of possible retries per-request.
+	  Do not touch this unless you know what you are doing.
```

Comment:
Maybe it would make more sense to have all of these with default values but allow them to be changed with an "expert mode" Kconfig which then allows them to be changed, like how bluetooth does it. That way, this message does not need repeating for every Kconfig